### PR TITLE
[v1.73] Graph context menu fixed + downgrade openshift-sdk to support OCP 4.12+

### LINKED ISSUE #284: Traffic graph context menu options do not redirect to the correct pages
### Describe the bug

Every node in the traffic graph has a context menu containing links to specific pages for that node (details, traffic, logs, etc.). However, none of the links redirect to the correct page within the OpenShift console.

image

### Expected Behavior

Each link in the context menu's node should redirect to correct page for that node.

### What are the steps to reproduce this bug?

1. Go to the traffic graph page
2. Click on the context menu of any node
3. Select any of the link options
4. Check that nothing happens

### Environment

- **OSSMC/Kiali version:** 1.73
- **OpenShift version:** 4.13
- **Istio version:** 1.18

### PR DESCRIPTION
**Pull request by ferhoyos**
### Describe the change
Backport of #261 to branch v1.73
Fixes #284 in branch v1.73

To avoid `yarn.lock` conflicts I have merged both changes in one PR (backport of #261 is quite simple, no test needed).

For the review, you can ignore changes in `src/kiali`, they have been copied from the Kiali repository.

### How to test the PR

1. Go to Graph page
2. Right click on any graph node and open context menu
3. Select at least two options:
  -- **Node graph**: Opens a new detailed graph. Check that back button works too.
  -- Any other option to go to details page

### CODE REVIEW

**Comment by ferhoyos** [comment]
Do not merge until OSSM 2.5.0 is released.

**Comment by ferhoyos** [comment]
OSSM 2.5 is released, so we can merge backports to v1.73 branch

File: plugin/package.json [hunk 1]
```diff
@@ -27,20 +27,19 @@
...
     "@patternfly/react-topology": "4.91.27",
-    "axios": "^0.21.4",
+    "axios": "^1.6.8",
     "bootstrap-slider-without-jquery": "10.0.0",
     "cy-node-html-label": "2.0.0",
     "cytoscape": "3.15.5",
     "cytoscape-canvas": "3.0.1",
     "cytoscape-popper": "1.0.7",
-    "dagre": "0.8.5",
+    "dagre": "0.8.2",
```
Comment (ferhoyos) [inline plugin/package.json:42]: `dagre` library is downgraded to match with `@patternfly/react-topology` dagre dependency and avoid library duplications

File: plugin/package.json [hunk 2]
```diff
@@ -126,13 +125,6 @@
     "webpack-cli": "^4.9.1",
     "webpack-dev-server": "^4.5.0"
   },
-  "resolutions": {
```
Comment (ferhoyos) [inline plugin/package.json:129]: I have reviewed all dependency resolutions and found them to be outdated. The previous versions of these libraries no longer exist to be overridden in the `yarn.lock` file. I haven't yet checked the Kiali repository, but it's likely that this change can also be applied there.

**Review by jshaughn — APPROVED** [review]
This looks good to me!

**Pull request merged**